assets/js/modules/analytics/datastore/tags.js

```diff
@@ -314,6 +318,9 @@ export const selectors = {
 	 * @return {?Object} Object with string `accountID` and boolean `permission` properties; `undefined` if not loaded.
 	 */
 	getTagPermission( state, propertyID ) {
+		if ( undefined === propertyID ) {
+			return undefined;
+		}
```

Comment:
Why do we need this? Passing `undefined` should crash into the `invariant` below IMO.